external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -1388,7 +1399,7 @@ fn error_format_impl(sp!(_, b_): &Type, subst: &Subst, nested: bool) -> String {
 fn error_format_impl_(b_: &Type_, subst: &Subst, nested: bool) -> String {
     use Type_::*;
     let res = match b_ {
-        UnresolvedError | Anything => "_".to_string(),
+        UnresolvedError | Anything | Void => "_".to_string(),
```

Comment:
I keep leaving this comment, but I don't really know where these strings will occur. But marking it again that maybe we want `!`